components/ResultsPage/EstimatedTotalItem.tsx

```diff
@@ -1,20 +1,35 @@
-import React from 'react'
+import React, { useState } from 'react'
 import { numberToStringCurrency } from '../../i18n/api'
 import { WebTranslations } from '../../i18n/web'
 import { Language } from '../../utils/api/definitions/enums'
 import { BenefitResult } from '../../utils/api/definitions/types'
 import { useTranslation } from '../Hooks'
+import { Modal } from './Modal'
+import Image from 'next/image'
 
 export const EstimatedTotalItem: React.VFC<{
   heading: string
   result: BenefitResult
   displayAmount: boolean
-}> = ({ heading, result, displayAmount }) => {
+  partner: boolean
+  maritalStatus
+}> = ({ heading, result, displayAmount, partner, maritalStatus }) => {
   const tsln = useTranslation<WebTranslations>()
+  const [modalOpen, setModalOpen] = useState(false)
+  const [modalPosition, setModalPosition] = useState({ x: 0, y: 0 })
   /*
     returns benefit name with from/de and proper article. ... french nuances.
   */
 
+  const openModal = (e) => {
+    setModalPosition({ x: e.clientX, y: e.clientY })
```

Comment:
See comment in the Modal component as well. Don't think tracking position is necessary.